src/library/simplified/extension/state-province-code.js

```diff
@@ -32,9 +32,9 @@ else if(utag.isPSR() && b['entity.packageSearch.packageSearchParameters.flightSe
         b["stateProvinceCode"] = b["stateProvinceCode"].split("(")[0].trim();
     }
 }
-else if(utag.isPSR_FHC() && b["entity.packageSearch.packageSearchParameters.flightSearchParameters.departureAirportCityState"] != undefined)
+else if((utag.isPSR_FH_Responsive() || utag.isPIS_FH() || utag.isPSR_FC() || utag.isPSR_FHC() || utag.isPIS_FHC() || utag.isPSR_HC() || utag.isPIS_HotelCar()) && b["entity.packageSearch.packageSearchParameters.flightSearchParameters.arrivalAirportCityState"] != undefined)
```

Comment:
this if statement is also hard to follow, please encapsulate in a method and name it accordingly.